src/__stories__/decorators.tsx

```diff
@@ -13,9 +14,9 @@ export const Tooltips = (storyFn: Function) => {
     if (path.length > 0) return [];
     return [
       {
-        summary: text('message', ''),
-        severity: number('severity', 40),
-        severityLabel: select('severityLabel', ['warn', 'anything-other-than-warn'], 'warn'),
+        message: text('message', ''),
+        severity: number('severity', DiagnosticSeverity.Warning),
+        range: {} as IRange,
```

Comment:
Can we make this a select with the values `[0, 1, 2, 3]`? Because I do want people to be able to preview the look for each level, and I also don't want it tied to TypeScript and its proprietary `enums`. (I don't want people to think they need to use TypeScript or import `@stoplight/types` in order to set the severity level.)

Reply:
Again, I guess this is just my ego / imagination that someone besides Stoplight might want to use Formtron without TypeScript/UI-Kit and I want the examples to be more approachable for this hypothetical person. 😆